### PR TITLE
feat(sidebar): add the ability to trigger action when sidebar is minimized, liv-6775

### PR DESCRIPTION
**Pull request by jinevo**

### CODE REVIEW

File: cypress/integration/spec-sidebar.ts [hunk 1]
```diff
@@ -41,6 +41,12 @@ describe('The Sidebar', () => {
       cy.getIframeContent('plugin', 'window').its('__messageReceived__').should('eq', messageToSend)
     })
 
+    // To activate when button is available on the app
+    // it('should be notified when the minimize button is clicked', function () {
+    //   cy.get('[data-testid="sidebar-header-minimize"]').click()
+    //   cy.getIframeContent('plugin', 'window').its('__onMinimizeTriggered__').should('eq', true)
+    // })
+
     describe('on close', () => {
```
Comment (Michaelvilleneuve) [inline cypress/integration/spec-sidebar.ts:51]: The test doesn't pass ?

**Review by Michaelvilleneuve — APPROVED** [review]
Just curious, what's the use case ?

File: cypress/integration/spec-sidebar.ts (hunk 1 above)
Comment (jinevo) [reply cypress/integration/spec-sidebar.ts:51]: Yes, since the button is new in the app, the test doesn't pass.

**Comment by jinevo** [comment]
> Just curious, what's the use case ?

This is purely an UX matter so that the users understand they can close safely the sidebar without killing the pending actions. The first use case is the Timer plugin which displays a small timer over the stage when the sidebar is minimze.

**Review by VincentGarreau — APPROVED** [review]

**Pull request merged**